packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -131,6 +131,10 @@ export interface ISemanticColors {
    * Background for success
    */
   successBackground: string;
+  /**
+   * The color for signals
```

Comment:
Please expand on what a signal is. Used for iconography indicating common states for things, etcetc.